javascripts/Start.js

```diff
@@ -3,12 +3,43 @@
 let start = false;
 let end = false;
 let pause = false;
+let request = require('request');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/gabholanda/module01-game/pullRequest?prid=8950836)
```suggestion
let request = require("request");
```